src/modules/Kdump.rb

```diff
@@ -385,10 +385,24 @@ def ReadKdumpSettings
     #
     # @return [Boolean] whether successful
     def update_initrd
+      # For CaaSP we need an explicit initrd rebuild before the
+      # first boot, when the root filesystem becomes read only.
+      rebuild_cmd = "/usr/sbin/tu-rebuild-kdump-initrd"
+      # part of transactional-update.rpm
+      update_initrd_with("if test -x #{rebuild_cmd}; then #{rebuild_cmd}; fi")
```

Comment:
in fact PBI said, that it should be enabled only if kdump is enabled, but I do not see in code that happen. Do you test that this is not called when kdump is not enabled?